lib/src/cmaf/ham/services/converters/hlsConverter.ts

```diff
@@ -1,6 +1,6 @@
 import { HlsMapper } from '../../mapper/HlsMapper.js';
 import { MapperContext } from '../../mapper/MapperContext.js';
-import type { Presentation } from '../../types/model/index.js';
+import type { Presentation } from '../../types/model';
```

Comment:
Is it okay do not importing index file? I remember having troubles building the project without importing index file